lighthouse-viewer/gulpfile.js

```diff
@@ -40,118 +57,113 @@ gulp.task('lint', () => {
 
 gulp.task('images', () => {
   return gulp.src('app/images/**/*')
-  .pipe(gulp.dest(`${config.dist}/images`));
+  .pipe(gulp.dest(`dist/images`));
 });
 
 gulp.task('concat-css', () => {
-  return gulp.src([
-    '../lighthouse-core/report/styles/report.css',
-    '../lighthouse-core/report/partials/*.css',
-    'app/styles/viewer.css',
-  ])
-  .pipe($.concat('viewer.css'))
-  .pipe(gulp.dest(`${config.dist}/styles`));
+  const reportCss = streamFromString(ReportGenerator.reportCss, 'report-styles.css');
+  const viewerCss = gulp.src('app/styles/viewer.css');
+
+  return streamqueue({objectMode: true}, reportCss, viewerCss)
```

Comment:
> is most of this to get around require() usage? Is there a reason you got ride of require? Kinda nice to have explicit deps defined. It was a pain to get working, but it was working :)

I guess it sort of is to get around it, but (I believe) it's necessary now. Viewer code is mixed with report renderer code in Viewer, so `require()` is a non-starter unless we want to convert all the report renderer code to using it too (which I don't believe we can).

The downside is, yeah, you have to look at `ReportGenerator` and this gulpfile to check what's included. On the other hand, we have less fragility in files trying to depend on things in node code, browserified node code, and/or native browser APIs.

Biggest complexity introduced in this file is that it delegates to `ReportGenerator` to pull in the report dependencies. e.g. instead of the old version including `'../lighthouse-core/report/styles/report.css'` in the `gulp.src`, it pulls it in as a string from `ReportGenerator.reportCss`. It makes the gulp stream nonsense more complicated, but it's nice from a DRY/maintenance perspective since any changes to report dependencies will automatically be pulled in here.